trecs/tests/test_measurements.py

```diff
@@ -182,6 +183,28 @@ def test_generic(self, timesteps=None):
         MeasurementUtils.test_generic_metric(ContentFiltering(), MSEMeasurement(), timesteps)
 
 
+class TestRMSEMeasurement:
```

Comment:
@sunnymatt can you add in some functionality to these tests for MSE and RMSE to make sure the returned value is actually a numeric value? That way, if we make some change that results in a NaN, Inf, or otherwise funky value, the test will catch it?